Server/mods/deathmatch/logic/CResource.h

```diff
@@ -256,8 +256,8 @@ class CResource : public EHS
     const std::string&          GetResourceDirectoryPath() const { return m_strResourceDirectoryPath; }
     const std::string&          GetResourceCacheDirectoryPath() const { return m_strResourceCachePath; }
 
-    std::list<CResourceFile*>& GetFiles() { return m_ResourceFiles; }
-    size_t                     GetFileCount() const noexcept { return m_ResourceFiles.size(); }
+    std::list<CResourceFile*>&       GetResourceFiles() noexcept { return m_ResourceFiles; }
```

Comment:
Non-const getter is usually a bad practice. Is it really needed here?

Reply:
I thought it may be modified somewhere. Better safe than sorry.
If it wont be modified anywhere then I can remove it

Reply:
I don't think that this getter will be useful. Even if so, it can be recreated very easy.